packages/manager/modules/pci/src/projects/project/private-registry/create/create.controller.js

```diff
@@ -0,0 +1,42 @@
+import get from 'lodash/get';
+
+export default class {
+  /* @ngInject */
+  constructor($stateParams, $translate, CucControllerHelper, privateRegistryService) {
+    this.$translate = $translate;
+    this.cucControllerHelper = CucControllerHelper;
+    this.privateRegistryService = privateRegistryService;
+    this.fromState = $stateParams.fromState;
+
+    this.projectId = $stateParams.projectId;
+    this.registry = {};
+  }
+
+  createRegistry() {
+    this.registry.region = 'GRA7'; // only GRA7 supported as of now
```

Comment:
There will be an issue for customers not having `GRA7` available . This cannot be a constant, an API call is needed to be able to get the regions available for a project

Reply:
For now i am getting this as a constant. The API for this is not there yet and will replace once api is available

Reply:
This issue has been discussed with API team, they will provide some solution to this may be a new API to get list of supported regions but now right now. We need to prod it in this state for now.

Reply:
I'm ok to let this go through to QA but imo this is an issue for prod